src/lib/core/observe-content/observe-content.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  Directive,
+  ElementRef,
+  NgModule,
+  ModuleWithProviders,
+  Output,
+  EventEmitter
+} from '@angular/core';
+
+@Directive({
+  selector: '[cdk-observe-content]'
+})
+export class ObserveContent {
```

Comment:
You should add an annotation to the class: `export class ObserveContent implements OnDestroy, AfterContentInit`.